android-components/components/feature/awesomebar/src/test/java/mozilla/components/feature/awesomebar/provider/SessionSuggestionProviderTest.kt

```diff
@@ -416,4 +416,34 @@ class SessionSuggestionProviderTest {
         assertTrue(suggestions.map { it.title }.contains("http://www.mobile.mozilla.org"))
         assertTrue(suggestions.map { it.title }.contains("https://www.mozilla.org/vpn"))
     }
+
+    @Test
+    fun `WHEN multiple tabs have the same url THEN provider returns a single suggestion for the matching input`() = runTest {
```

Comment:
nit, just a small name change that might fit better:
```suggestion
    fun `GIVEN multiple tabs have the same url WHEN user searches the same url THEN provider returns a single suggestion for the matching input`() = runTest {
```